routing/road_access_serialization.hpp

```diff
@@ -115,15 +130,29 @@ class RoadAccessSerializer final
   }
 
   template <typename Source>
-  static void DeserializeOneVehicleType(Source & src, RoadAccessTypesMap & m)
+  static void DeserializeOneVehicleType(Source & src, RoadAccessTypesFeatureMap & mf,
+                                        RoadAccessTypesPointMap & mp)
   {
-    m.clear();
+    mf.clear();
+    mp.clear();
     for (size_t i = 0; i < static_cast<size_t>(RoadAccess::Type::Count); ++i)
     {
       std::vector<Segment> segs;
       DeserializeSegments(src, segs);
       for (auto const & seg : segs)
-        m[seg] = static_cast<RoadAccess::Type>(i);
+      {
+        if (seg.GetSegmentIdx() == 0)
```

Comment:
Где-то здесь стоит дописать что-то типа
```
// An earlier version allowed blocking any segment of a feature (or the entire feature
// by providing a wildcard segment index).
// At present, we either block the feature entirely or block any of its road points. The
// the serialization code remains the same, although its semantics changes as we now
// work with point indices instead of segment indices.
```

Reply:
у нас не было ни одной mwm-ки сгенерировано с не wildcard сегментами, поэтому мне кажется, что писать коммент о том что был когда-то такой план -- только запутывать

Reply:
Я имею в виду, что код был для этого предназначен. В частности, именование переменных. Сейчас в десериализаторе segment то, segment это, а на выходе две мапки без сегментов.

Reply:
ok, done